tests/kuttl/tests/keystone_scale/03-assert.yaml

```diff
@@ -0,0 +1,53 @@
+#
+# Check for:
+#
+# - 1 KeystoneAPI CR
+# - 3 Pods for KeystoneAPI CR
+#
+
+apiVersion: keystone.openstack.org/v1beta1
+kind: KeystoneAPI
+metadata:
+  finalizers:
+  - KeystoneAPI
+  name: keystone
+  namespace: openstack
+spec:
+  adminProject: admin
+  adminRole: admin
+  adminUser: admin
+  containerImage: quay.io/tripleozedcentos9/openstack-keystone:current-tripleo
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  databaseUser: keystone
+  debug:
+    bootstrap: false
+    dbSync: false
+    service: false
+  passwordSelectors:
+    admin: AdminPassword
+    database: KeystoneDatabasePassword
+  preserveJobs: false
+  region: regionOne
+  replicas: 3
+  resources:
+    requests:
+      cpu: "1.0"
+      memory: 500Mi
+  secret: osp-secret
+status:
+  apiEndpoint:
+    admin: http://keystone-admin-openstack.apps-crc.testing
+    internal: http://keystone-internal-openstack.apps-crc.testing
+    public: http://keystone-public-openstack.apps-crc.testing
+  databaseHostname: openstack
+  readyCount: 3
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keystone
+spec:
+  replicas: 3
```

Comment:
This only asserts that keystone operator changed the replica count on the deployment to 3. But it does not wait to see if 3 replicas are actually possible to start up successfully. To assert that:
```yaml
status:
    availableReplicas: 3
```
will wait for the 3 Pods to be ready.

Reply:
That makes a lot of sense, added. Thanks for the suggestion!